requirements/dev.txt

```diff
@@ -9,6 +9,7 @@ django-extensions==1.7.8
 django-silk==1.0.0
 django-spaghetti-and-meatballs==0.2.2
 flake8==3.0.4
+google-compute-engine==2.8.3
```

Comment:
Why do we need this?

Reply:
Earlier it threw that this module isn't found. I am going to remove this line only.